pkg/controller/master/image/vm_image_controller.go

```diff
@@ -142,10 +149,14 @@ func (h *vmImageHandler) checkImage(image *harvesterv1.VirtualMachineImage) (*ha
 
 func (h *vmImageHandler) createBackingImageAndStorageClass(image *harvesterv1.VirtualMachineImage) (*harvesterv1.VirtualMachineImage, error) {
 	if err := h.createBackingImage(image); err != nil && !errors.IsAlreadyExists(err) {
-		return nil, err
+		logrus.Errorf("failed to create backing image in image %s/%s, err: %v", image.Namespace, image.Name, err)
```

Comment:
```suggestion
		logrus.WithError(err).WithFields(logrus.Fields{
			"namespace": image.Namespace,
			"name":      image.Name,
		}).Error("failed to create backing image for vmimage")
```

Reply:
Thanks, I've changed all error log format in vm_image_controller to what you have suggested in https://github.com/harvester/harvester/pull/5498/commits/3156aa6a01658b01bd673be6e429e2fce9733d4b